wdl-analysis/src/queue.rs

```diff
@@ -735,3 +886,91 @@ pub(crate) fn format_panic_payload(payload: &Box<dyn std::any::Any + Send>) -> S
         "unknown panic payload".to_string()
     }
 }
+
+/// Finds an identifier token at the specified byte offset in the CST.
+fn find_identifier_at_offset(node: &SyntaxNode, offset: usize) -> Result<Option<String>> {
```

Comment:
Does this need to return a `String` or can it return a `&str` (skip the allocation).